quality/pmd/android.gradle

```diff
@@ -10,7 +10,7 @@ pmd {
   toolVersion = '5.8.1' // see https://github.com/pmd/pmd/releases
 }
 
-check.dependsOn 'pmd'
+assemble.dependsOn 'pmd'
```

Comment:
This change means that PMD will no longer run when running the `./gradlew check` command, right? Is there a different solution you can think of?

Reply:
I totally agree with @corycaywood